### PR TITLE
add hacktoberfest 2018 with auth0

### PR DESCRIPTION
**Pull request by lukeoliff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,13 @@
 
 ## Coding for Beginners
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
+-
```
Comment (OtacilioN) [inline README.md]: Hey @lukeoliff I am just not sure about this line, can you fix it?

File: README.md [hunk 2]
```diff
@@ -16,11 +16,13 @@
 
 ## Coding for Beginners
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
+- 
 
 ## Meetups and Events
 * **Hacktoberfest @ Sorbonne Université** in Paris (France), the 13th of October ([Eventbrite](https://www.eventbrite.ca/e/billets-hacktoberfest-sorbonne-universite-2018-50269271745))
  - [Hacktoberfest em Recife! 🇧🇷](https://www.sympla.com.br/hacktoberfest-em-recife__367675), the 6th of October
  - [Hacktoberfest in São Luís 🇧🇷](https://hacktoberfest-in-slz.github.io/pagina/), the 6th of October
+- [Celebrate Hacktoberfest 2018 with Auth0!](https://auth0.com/blog/celebrate-hacktoberfest-with-auth0/), online event all October
```
Comment (OtacilioN) [inline README.md]: Oh, online events are awesome! Thanks for contributing!

File: README.md (hunk 1 above)
Comment (lukeoliff) [reply README.md]: Oh, what happened there! 😲I'll fix it!

**Comment by lukeoliff** [comment]
Changes made :)

**Comment by OtacilioN** [comment]
Awesome, thanks for contributing!

**Pull request merged**